README.md

```diff
@@ -443,19 +443,19 @@ Note that you can use ``add_ivars!`` and ``add_bvars!`` to add multiple integer
 ```julia
 using JuMP, Gurobi
 
-m = Model(solver=GurobiSolver())
+model = Model(with_optimizer(Gurobi.Optimizer))
 
-@variables(m,begin
+@variables(model,begin
```

Comment:
Missing space after comma

Reply:
There was no space previously.

Reply:
These suggestions are all coming from the JuMP style guide. It's worth a read if you haven't already: https://pkg.julialang.org/docs/JuMP/DmXqY/0.19.0/style/#Whitespace-1

Reply:
I'm objecting to the way a bug-fix pull request has been turned into an opportunity to update the existing code's style. I'm happy to do the bug fix and happy to cover things I've missed, but modifying parts of the code I didn't write to fit style conventions I don't know is not something I really want to do.

Reply:
@r-barnes This is understandably frustrating, but don't take these comments as a criticism of your changes. Our community has a convention (which is mentioned in the style guide) to fix style issues in code when it's touched, i.e., leaving the code tidier than when you found it. This is a sustainable way to propagate the style changes, reduce entropy in the codebase, and reduce the burden on maintainers. Your contributions are appreciated and will hopefully go more smoothly the next time around.

As an aside, this manual process will disappear once Julia has a proper linter that we can run as a CI pass.

Reply:
Sounds good, thanks.